docs/en/observability/monitor-azure.asciidoc

```diff
@@ -12,28 +12,154 @@ using Elastic Observability: Logs and Metrics.
 
 You'll learn how to:
 
-- Set up an Azure service principal.
-- Ingest metrics using the {metricbeat-ref}/metricbeat-module-azure.html[{metricbeat}
-Azure module] and view those metrics in {kib}.
-- Export Azure activity logs through Event Hubs.
-- Ingest logs using the {filebeat-ref}/filebeat-module-azure.html[{filebeat}
-Azure module] and view those logs in {kib}.
+- Create an {es} resource in the Azure portal.
+- Ingest logs using the native integration and view those logs in Kibana.
```

Comment:
We should be explicit and call this `Azure platform logs`

Reply:
Thank you for the review Osman. I am a bit confused. Can you confirm that the following is your suggestion:

"Ingest Azure platform logs using the native integration and view those logs in Kibana."

Reply:
Correct. The reason is because just saying "logs" is too generic since there are lots of different kinds of logs throughout the Azure platform.